<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>231</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>makeAutoPlot</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>556</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLabel" name="CMLabel">
      <property name="text">
       <string>Cryomodule:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Display:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_4">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Start ST or AV:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="DispComboBox"/>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="GoButton">
      <property name="text">
       <string>Go!</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="STradioButton">
        <property name="text">
         <string>StripTool</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="AVradioButton">
        <property name="text">
         <string>Archive Viewer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="CavComboBox"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="CavLabel">
      <property name="text">
       <string>Cavity&lt;br&gt;  if needed:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QLabel" name="TitleText">
      <property name="maximumSize">
       <size>
        <width>1677715</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Choose a CM, a group of signals, Striptool or Archive Viewer, then click Go!</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="PyDMLabel" name="DescLabel">
      <property name="toolTip">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="CMComboBox"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Choose:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
